src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/CaseHearingFunctions.java

```diff
@@ -100,26 +101,40 @@
     };
 
     static String getSelectedCourt(Map<String, Object> mapOfCaseData) {
-        switch ((String) mapOfCaseData.get(REGION)) {
+        return getSelectedCourt(mapOfCaseData, "");
+    }
+
+    static String getSelectedCourt(Map<String, Object> mapOfCaseData, String fieldNamePrefix) {
```

Comment:
looks like this might need some test coverage

Reply:
Added tests.